src/plugins/data/common/index_patterns/fields/index_pattern_field.ts

```diff
@@ -20,40 +20,27 @@
 import { i18n } from '@kbn/i18n';
 import { KbnFieldType, getKbnFieldType } from '../../kbn_field_types';
 import { KBN_FIELD_TYPES } from '../../kbn_field_types/types';
-import { FieldFormat } from '../../field_formats';
 import { IFieldType } from './types';
-import { OnNotification, FieldSpec } from '../types';
-
-import { IndexPattern } from '../index_patterns';
+import { FieldSpec, IndexPattern } from '../..';
+import { FieldTypeUnknownError } from '../errors';
 
 export class IndexPatternField implements IFieldType {
   readonly spec: FieldSpec;
   // not writable or serialized
```

Comment:
nit: maybe this comment can go now too? since i assume it's for the line you just removed

Reply:
applies to displayName and kbnFieldType